workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/pages/AllScreens.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayLayout from '@clayui/layout';
+import {ClayVerticalNav} from '@clayui/nav';
+import {ReactNode} from 'react';
+
+import {useHash} from '../hooks/useHash';
+import {ScreenType} from '../types';
+import TicketsDashboard from './TicketsDashboard';
+import TicketsOverview from './TicketsOverview';
+
+const ROUTES = [
+	{
+		element: <TicketsDashboard screenType={ScreenType.INTEGRATED} />,
+		href: '#dashboard',
+		label: 'Dashboard',
+	},
+	{
+		element: <TicketsOverview />,
+		href: '#overview',
+		label: 'Tickets Overview',
+	},
+];
+
+const HREF_COMPONENT_MAP: {[key: string]: ReactNode} = {};
+
+ROUTES.forEach((route) => {
+	HREF_COMPONENT_MAP[route.href] = route.element;
+});
```

Comment:
This is odd. Better to just add this to the static object because this is run immediately when the file is loaded, so there is no need to run this logic since we already know it's value.